frontend/src/Pages/Home/InputForm/InputForm.tsx

```diff
@@ -167,22 +214,60 @@ const InputForm = () => {
           error={formik.touched.city && Boolean(formik.errors.city)}
           helperText={formik.touched.city && formik.errors.city}
         />
-        <TextField
-          id="relevantSkills"
-          className="input-form"
-          name="relevantSkills"
-          label="Relevant Skills"
-          value={formik.values.relevantSkills}
-          onChange={formik.handleChange}
-          onBlur={formik.handleBlur}
+        <FormControl
+          className="input-form hide-arrow"
           error={
             formik.touched.relevantSkills &&
             Boolean(formik.errors.relevantSkills)
           }
-          helperText={
-            formik.touched.relevantSkills && formik.errors.relevantSkills
-          }
-        />
+        >
+          <InputLabel id="relevantSkillsLabel">Relevant Skills</InputLabel>
+          <Box sx={{ display: 'flex', alignItems: 'center' }}>
+            <Select
```

Comment:
add `name="relevantSkills"` here

Reply:
included name="relevantSkills" in new AutoComplete state.